include/alpaka/kernel/Traits.hpp

```diff
@@ -215,15 +215,26 @@ namespace alpaka
                 static_assert(std::is_same_v<Result, void>, "The TKernelFnObj is required to return void!");
             }
         };
+
+        // asserts that T is trivially copyable. We put this in a separate function so we can see which T would fail
+        // the test, when called from a fold expression.
+        template<typename T>
+        inline void assertKernelArgIsTriviallyCopyable()
+        {
+            static_assert(
+                std::is_empty<T> || std::is_trivially_copyable_v<T>,
+                "The kernel argument T must be trivially copyable!");
```

Comment:
```suggestion
            static_assert(
                std::is_empty_v<T> || std::is_trivially_copyable_v<T>,
                "The kernel argument T must be trivially copyable!");
```
Sorry, I messed this up in my suggestion!